### PR TITLE
Seperate ineligible intake errors

### PR DESCRIPTION
**Pull request by shanear**
For demo testing use,
11555555 for "Appeal is at the board"
12555555 for "No active appeals"

connects #3775

### CODE REVIEW

File: app/models/appeal.rb [hunk 1]
```diff
@@ -208,7 +208,7 @@ def hearing_pending?
   end
 
   def eligible_for_ramp?
-    status == "Advance"
+    status == "Advance" || status == "Remand"
```
Comment (shanear) [inline app/models/appeal.rb]: this is based on a recent change in ramp eligibility by AMO. @cmgiven want to confirm?

Comment (nickheiner-usds) [reply app/models/appeal.rb]: I will 100% trust @cmgiven to validate the business logic here. :smile:

Comment (cmgiven) [reply app/models/appeal.rb]: That's correct.

File: spec/models/appeal_spec.rb [hunk 2]
```diff
@@ -783,11 +783,16 @@
   context "#eligible_for_ramp?" do
     subject { appeal.eligible_for_ramp? }
 
-    context "is false if status is not advance" do
-      let(:appeal) { Generators::Appeal.build(vacols_id: "123", status: "Remand") }
+    context "is true if status is not advance or remand" do
```
Comment (nickheiner-usds) [inline spec/models/appeal_spec.rb]: Isn't this the opposite of what `eligible_for_ramp?` does?

```rb
  def eligible_for_ramp?
    status == "Advance" || status == "Remand"
  end
```

Comment (nickheiner-usds) [reply spec/models/appeal_spec.rb]: And isn't the test actually checking for the method to return `false`?

File: app/models/appeal.rb (hunk 1 above)
Comment (nickheiner-usds) [reply app/models/appeal.rb]: Also, does this mean that only AOD or Remand appeals are eligible? I thought it was a broader set than that.

Comment (shanear) [reply app/models/appeal.rb]: "Advance" means the appeal is not remanded and at a regional office

**Pull request merged**